examples/pbc/lasci_gamma_point.py

```diff
@@ -0,0 +1,72 @@
+import numpy
+from pyscf import gto, scf
+from pyscf import mcscf
+from mrh.my_pyscf.mcscf.lasscf_o0 import LASSCF
+from mrh.my_pyscf.tools.molden import from_lasscf
+
+'''
+Here I am doing first gamma point LASCI calculation
+For reference I did molecular calculation, periodic calculation
+with large unit cell (supercell) and at gamma point should be 
+equals to the molecular values
+Outcome of these calculations
+1. HF is converging to molecular value, so is the casci
+2. LASCI is not converging (Have to find out why !)
+
+Molecular calculations are done without density fitting, but
+for periodic calculations FFTDF is by default on.
+Probably that's the reason of slow convergence!
+'''
+
+# Molecular Calculation
+mol = gto.M(atom = '''
+        H         -6.37665        2.20769        3.00000
+        H         -5.81119        2.63374        3.00000
+    ''',
+    basis = 'sto3g',
+    verbose = 1,max_memory=10000)
+
+mf = scf.RHF(mol)
+memf = mf.kernel()
+
+mc = mcscf.CASCI(mf, 2, 2)
+mecasci = mc.kernel()[0]
+
+las = LASSCF(mf, (2,), (2, ))
+mo0 = las.localize_init_guess((list(range(2)), ))
+melasci = las.lasci_()
+
+del mol, mf, mc, las
+
+# Periodic Calculation
+from pyscf.pbc import gto, scf
+
+def cellObject(x):
+    cell = gto.M(a = numpy.eye(3)*x,
+    atom = '''
+        H         -6.37665        2.20769        3.00000
+        H         -5.81119        2.63374        3.00000
+    ''',
+    basis = 'sto3g',
+    verbose = 1, max_memory=10000)
+
+    mf = scf.RHF(cell)
+    emf = mf.kernel()
+
+    mc = mcscf.CASCI(mf, 2, 2)
+    ecasci = mc.kernel()[0]
+
+    las = LASSCF(mf, (2,), (2, ))
+    mo0 = las.localize_init_guess((list(range(2)), ))
```

Comment:
For the record, this line doesn't do anything, since you don't pass the output to the `lasci_` call next line.

Reply:
Correct. 

I will correct it. but it won't change it.